internal/backend/rclone/internal_test.go

```diff
@@ -23,7 +23,9 @@ func TestRcloneExit(t *testing.T) {
 		return
 	}
 	rtest.OK(t, err)
-	defer be.Close()
+	defer func() {
+		rtest.OK(t, be.Close())
```

Comment:
That error should be ignored as rclone is killed in line 30

Reply:
Agreed.